.github/workflows/cml.yaml

```
@@ -1,5 +1,8 @@
 name: CML Report
-on: push
+on: [push, pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
```

---

Actually not sure this is right, as it relies on names rather than SHAs (so susceptible to race conditions).

[`github` context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) | Name | SHA
-|-|-
source | `head_ref`[^pr], `ref_name`[^diff] | `event.pull_request.head.sha`[^pr], `sha`[^diff]
target | `base_ref`[^pr] | -

[^diff]: different values for `push` & for `pr`
[^pr]: only defined for `pr`

```suggestion
  group: ${{ github.workflow }}-${{ event.pull_request.head.sha || github.sha }}
```